roles-in-tech/qa.md

```diff
@@ -2,89 +2,105 @@
 
 ### Projected Time
 
-Example: 30-45 minutes
-- Lesson: ? min
-- Guided Practice: ?-? min
-- Independent Practice: ? min
-- Check for Understanding: ?-? min
+- Lesson: 20 min
+- Guided Practice: 10 min
+- Independent Practice: 12 min
+- Check for Understanding: 15 min
 
 ### Prerequisites
 
-Here are links to lessons that should be completed before this lesson:
-
-- Example: [Git version control](version-control/git-version-control/git-version-control.md)
+None
 
 ### Motivation
 
-Here's why this topic should be learned. Strengthen this section by including a company that uses this skillset, and a screenshot example if possible.
+- When your customers think of your brand, certain images, feelings, and expectations come into their minds. A good QA system is a necessary component of your business success, especially nowadays, when unhappy customers are just a click away from multiple social media outlets. 
+- QA engineers, also known as quality assurance engineers, play an important role in the manufacturing process. They are responsible for making sure all manufactured products meet company and industry quality standards.
+- That’s why the best teams have QAs working together with others; they can combine their skills in the pursuit of quality products.
 
 ### Objectives
 
 **Participants will be able to:**
 
-- leave this lesson knowing this
-- and how to do this on their own
-- Even more things
-- Even more things
+- Understand the role of a quality assurance engineer in a company
+- Decide whether they are suitable for a future as a quality assurance engineer and if so the steps to be taken in-order to become one.
 
 ### Specific Things To Teach
 
-- Focus on this specific talking point
-- and this
-	- This is a sub-thing about the thing
-- Even more things about the things
-- Even more things about the things
+- Who is a quality assurance engineer
+- Duties of a quality assurance engineer
+- Traits of a quality assurance engineer
+- Steps to take for a career as a quality assurance engineer
 
 ### Materials
 
-- [Example video (10 min)](https://example.com) - Write a very short description of this resource.
-- [Other example article(20 min read)](https://otherexample.com) - Answer the 3 questions at the end.
-(Be sure to explain how to use these materials in directions in one of the sections below (ie: lesson, guided practice).  If you are not assigning them somehow, they should be placed under Supplemental Materials at the bottom.) 
+- [Quality assurance engineering slideshow](https://docs.google.com/presentation/d/17PG75lHEEOaoNM6OuAUeoaLa7Hf14tJPiwwdy5I-G-c/edit#slide=id.g3509c77696_0_9)
```

Comment:
Are all of the links assigned below listed here?

Reply:
If any of these are just for their reference, and are not assigned, please put them under a different header, "supplemental materials".